src/main/java/org/somerville/swag/data/source/MyFileWriter.java

```diff
@@ -0,0 +1,9 @@
+package org.somerville.swag.data.source;
+
+import org.somerville.swag.data.exception.FileWriterException;
+
+public interface MyFileWriter {
```

Comment:
I really like this way of implementing the File writer, should make life easier when making Stub tests